bootstrap

```diff
@@ -141,7 +141,11 @@ SVER=${LVER%%.*}
 ./$VENV/bin/pip install --upgrade pip
 
 # Ensure setuptools is installed
-./$VENV/bin/pip install setuptools --upgrade
+if [ "$PYTHON" = "python2" ]; then
```

Comment:
actually, `"$SVER" = "2"` may work better, since one can use PYTHON=python2.7 for example.